multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1537,6 +1606,11 @@ def extract_embedding(
         data
             The data to extract embeddings for. Should contain same column names as training dataset and
             follow same format (except for the `label` column).
+        return_masks
+            Whether to return the masks indicating whether the corresponding features are valid.
+            If one sample's some column have empty input, then the sample has mask 0 for that column.
```

Comment:
Consider to change to 

```python
Return the masks that indicate the sample contains missing values. If a sample contains missing values, the features can be invalid and we use normalized constant vectors as the feature.
```

Reply:
The mask is feature column-wise, not sample-wise. A sample may have two feature columns `image1` and `image2`. We extract features for `image1` and `image2` separately. Maybe the image of `image1` is missing, but `image2` has a valid image. Then `image1` of this sample has mask 0 while `image2` of this sample has mask 1.

Reply:
Okay, the description is not very easy to understand. Also, do we really need it?

Reply:
Users may need this information in some cases. Returning masks is optional (default False).

Reply:
We may need to improve the documentation. Currently it's difficult to understand this flag.

Reply:
Refined.